### DISCUSSION TITLE
space before month in datePicker

### DISCUSSION THREAD

**Discussion by vladdoletskiy** [adobe/react-spectrum#3386] [Q&A]
Hello, you have bad space before the month in datePicker. In your doc - i see too. 
https://react-spectrum.adobe.com/react-spectrum/DatePicker.html

**Comment by snowystinger** [comment]
Please see https://github.com/adobe/react-spectrum/issues/3355

**Reply by vladdoletskiy** [reply]
near month you have 2 spaces.

**Reply by vladdoletskiy** [reply]
![Screenshot_39](https://user-images.githubusercontent.com/62791013/183506327-6011ad32-bba2-466c-9673-ca53828f8fc0.png)
@snowystinger check please it space.
